.github/workflows/pre-merge.yaml

```
@@ -40,7 +40,7 @@ jobs:
       - name: Build detekt
         uses: gradle/gradle-build-action@3fbe033aaae657f011f88f29be9e65ed26bd29ef # tag=v2
         with:
-          arguments: build -x detekt publishToMavenLocal
+          arguments: build -x detekt publishToMavenLocal -PenablePTS=${{ github.event_name != 'push' }}
```

```suggestion
          arguments: build -x detekt publishToMavenLocal -PenablePTS=${{ github.event_name != 'push' }}
```
nice iterations, it looks pretty simple now and more natural for anyone who reads it.

I was thinking about `!= push` vs `== pull_request`, because of our current PR title. But then we kind of want to "disable it for main" so that it runs stricter on main, and uses PTS everywhere else (PRs and potential other stuff that we might add as a trigger). ⚖ 🤔

---

Agree! Done.